vsts/scripts/publishSdksFromDevToProdStorageAccount.sh

```diff
@@ -51,12 +51,31 @@ function copyBlob() {
 
 function copyPlatformBlobsToProd() {
```

Comment:
We have another script for publishing SDKs to dev, just a reminder in case we also need to make any changes there.

Reply:
@qianz2 Correct me if I'm wrong, but I think that SDKs get to dev by the following:
1. PlatformBinaries job is made with `../templates/_platformBinariesTemplate.yml`
2. `../templates/_platformBinariesTemplate.yml` invokes the `./build/buildPlatformBinaries.sh` script
3. `buildPlatformBinaries.sh` invokes the build{Platform}.sh scripts in the `platforms/` dir that I have modified to support this change
4. Those scripts place the SDKs in a local `artifacts/` directory
5. `/scripts/publishFilesToAzureStorage.sh` is run by the platform binaries job, which finds the contents of the `artifacts/` directory and publishes them

If all that is correct, then I believe that everything has been updated appropriately. I won't know for sure until I merge though, i think